pkg/crc/config/settings.go

```diff
@@ -145,14 +139,14 @@ func GetPreset(config Storage) preset.Preset {
 }
 
 func defaultNetworkMode() network.Mode {
-	if version.IsInstaller() {
+	if runtime.GOOS != "linux" {
 		return network.UserNetworkingMode
 	}
```

Comment:
This is the hunk which is doing what the commit log describes (using the same default networking mode on release/non-release builds).